src/windows/editor_object_selection.c

```diff
@@ -1888,7 +1903,7 @@ static rct_string_id get_ride_type_string_id(const ObjectRepositoryItem * item)
 	for (int i = 0; i < 3; i++) {
 		uint8 rideType = item->RideType[i];
 		if (rideType != 255) {
-			result = 2 + rideType;
+			result = STR_RIDE_NAME_SPIRAL_ROLLER_COASTER + rideType;
```

Comment:
For the descriptions as well. But I found the 'fallback ride types' a bit odd, so I decided to skip that for now. I'm fairly sure I haven't been able to catch all strings anyway.